### ISSUE TITLE
Use of unknown attribute "data_reuse_license" in views/javascript?

### ISSUE THREAD

**Issue by nicolasfranck** [LibreCat/LibreCat#942]
These views assume existence of publication attribute `data_reuse_license`:

* https://github.com/LibreCat/LibreCat/blob/master/views/publication/tab_details.tt#L134
* https://github.com/LibreCat/LibreCat/blob/master/views/backend/generator/fields/jquery_tab1.tt#L67

But this field is not known in the publication schema.
Is this an old remnant of the past?

**Comment by petrakohorst** [comment]
Yes, these are old research_data remains. We had them fixed in our local layer.

I'll prepare a PR for this.

**Comment by petrakohorst** [comment]
How do we generally want to handle licenses in LibreCat?

The changes in ```jquery_tab1.tt``` will have an effect on whether selecting a license is mandatory once an (oa-)file has been uploaded or not. So far - in LibreCat - a license is only mandatory for research_data records with files.

In our local installation, however, we have made licenses mandatory for all types with oa-file.

What should I implement here?
